src/mesh_sandbox/handlers/inbox.py

```diff
@@ -241,7 +241,11 @@ async def _get_inbox_messages(
     ) -> tuple[list[Message], Optional[dict]]:
 
         messages = (
-            await self.messaging.get_inbox_messages(mailbox.mailbox_id)
+            sorted(
+                await self.messaging.get_inbox_messages(mailbox.mailbox_id),
+                key=lambda message: message.created_timestamp,
+                reverse=True,
+            )
             if rich
             else await self.messaging.get_accepted_inbox_messages(mailbox.mailbox_id)
```

Comment:
should sort be applied here too ? ( or does that sort internally ? )

Reply:
i think the inbox sorts ascending: https://github.com/NHSDigital/mesh/blob/develop/src/shared/store.py#L289

and the rich inbox sorts descending: https://github.com/NHSDigital/mesh/blob/develop/src/shared/store.py#L407